mix.exs

```diff
@@ -31,9 +31,12 @@ defmodule Fields.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:argon2_elixir, "~> 2.1.2"},    # password hashing
-      {:ecto, "~> 3.2.5"},             # ecto types
-      {:html_sanitize_ex, "~> 1.3"}, # strip noise from html field
+      # password hashing
```

Comment:
increases versions of dependencies and formats in-line comments to separate lines. 👍